src/command_modules/azure-cli-acs/setup.py

```diff
@@ -14,7 +14,7 @@
     logger.warn("Wheel is not available, disabling bdist_wheel hook")
     cmdclass = {}
 
-VERSION = "2.0.20"
+VERSION = "2.0.21"
```

Comment:
I *think* we're supposed to leave this alone and let the maintainers increment it--at least I had a previous PR where that was the case.

Reply:
Oh ok, I did it because it failed the CI. Do you want me to put it back version .20?

Reply:
There is a process change recently that if yours is the first one since the latest release, you should bump up the version here and in the HISTORY.RST; otherwise CI will fail